package.json

```diff
@@ -10,6 +10,7 @@
     "prop-types": "^15.8.1",
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229077587-5beb8270-561e-457c-a21e-48146606c9ec.png)

- [ ] It would be nice if you update this section as well shown in the image above ⬆️ to make your readme file descriptive and professional 😊